pcsx2/x86/iR3000A.cpp

```diff
@@ -941,10 +941,16 @@ static void iPsxBranchTest(u32 newpc, u32 cpuBranch)
 		xADD(eax, blockCycles);
 		xMOV(ptr32[&psxRegs.cycle], eax); // update cycles
 
+		if ((psxHu32(HW_ICFG) & (1 << 3))) // if were in PS1 mode adjust ee accordingly.
+		{
+			psxEEmultiplier = 8.71;
+		}
+
 		// jump if iopCycleEE <= 0  (iop's timeslice timed out, so time to return control to the EE)
-		xSUB(ptr32[&iopCycleEE], blockCycles*8);
+		xSUB(ptr32[&iopCycleEE], blockCycles*static_cast<int>(psxEEmultiplier));
```

Comment:
You should explain why you "truncate" the multiplier instead of rounding. And why you do it before the multiplication.
i.e.
why not ?
```
static_cast<int>(blockCycles* psxEEmultiplier);
```
or
```
static_cast<int>(round(blockCycles* psxEEmultiplier));
```